test/spear_test.exs

```diff
@@ -919,28 +919,15 @@ defmodule SpearTest do
       assert Spear.stream!(c.conn, c.stream_name) |> Enum.to_list() == []
     end
 
-    # deadlines do not appear to be working no matter what timestamp I put on
-    # or how I interleave the messages
-    # @tag compatible("~> 21.6")
-    @tag :skip
+    @tag compatible(:nightly)
```

Comment:
fixed https://github.com/EventStore/EventStore/pull/3138

nice to be able to test it now in CI and see it pass :rocket: